modules/statics/src/coins.ts

```diff
@@ -1342,6 +1342,29 @@ export const coins = CoinMap.fromCoins([
   erc20('xtzbull', '3X Long Tezos Token', 18, '0x8af17a6396c8f315f6b6dbc6aa686c85f9b3e554', UnderlyingAsset.XTZBULL),
   erc20('zecbear', '3X Short Zcash Token', 18, '0x78a8c84b4c23563be4518e7045016d3170130823', UnderlyingAsset.ZECBEAR),
   erc20('zecbull', '3X Long Zcash Token', 18, '0xd437d88153daef4784cacd2084b1d8cc2d3312b8', UnderlyingAsset.ZECBULL),
+  erc20('cow', 'CoW Protocol Token', 18, '0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab', UnderlyingAsset.COW),
+  erc20('cpool', 'Clearpool', 18, '0x66761fa41377003622aee3c7675fc7b5c1c2fac5', UnderlyingAsset.CPOOL),
+  erc20('eoserc20', 'EOS ERC20 Token', 18, '0x86fa049857e0209aa7d9e616f7eb3b3b78ecfdb0', UnderlyingAsset.EOS, [
```

Comment:
Should `EOS ERC20 Token` be `EOS` to match https://etherscan.io/address/0x86fa049857e0209aa7d9e616f7eb3b3b78ecfdb0?

Reply:
I'm guessing it's this way to differ it from the eos coin?

Reply:
@mmcshinsky-bitgo great question lol. I was debating with myself on this and am not sure about the conventions. I used this to help differentiate between the actual EOS blockchain.

I don't feel too strongly about it one way or the other.

Reply:
One of the biggest issues we'll see long term are these name collisions with multiple coins and their tokens and our architecture giving both first class citizenship. Would `erc20:eos` be a good way to dial in on the coin it belongs to for the name collision?

Reply:
We use the convention `<chain>:<token>` for everything except ERC20s, e.g. `eos:check`, `eos:usdt`, `hbar:usdc`
ERC20s is just `<token>`.

going to leave as is